console/src/components/Services/RemoteSchema/Edit/View.js

```diff
@@ -1,28 +1,60 @@
 import React from 'react';
-
 import CommonTabLayout from '../../../Common/Layout/CommonTabLayout/CommonTabLayout';
 import tabInfo from './tabInfo';
-import Tooltip from 'react-bootstrap/lib/Tooltip';
-import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import { push } from 'react-router-redux';
-
 import {
   fetchRemoteSchema,
   RESET,
   getHeaderEvents,
 } from '../Add/addRemoteSchemaReducer';
-
 import { VIEW_REMOTE_SCHEMA } from '../Actions';
 import ReloadRemoteSchema from '../../Settings/MetadataOptions/ReloadRemoteSchema';
-
 import { appPrefix } from '../constants';
-
-import { NotFoundError } from '../../../Error/PageNotFound';
-
 import globals from '../../../../Globals';
+import styles from '../RemoteSchema.scss';
+import ToolTip from '../../../Common/Tooltip/Tooltip';
 
 const prefixUrl = globals.urlPrefix + appPrefix;
 
+const RSHeadersDisplay = ({ data }) =>
+  data.length > 0 ? (
+    <tr>
+      <td>Headers</td>
+      <td>
+        {data &&
+          data
+            .filter(header => !!header.name)
+            .map((header, index) => [
+              <tr key={header}>
+                <td>
+                  {header.name}:{' '}
+                  {header.type === 'static'
+                    ? header.value
+                    : '<' + header.value + '>'}
+                </td>
+              </tr>,
+              index !== data.length - 1 ? <hr /> : null,
+            ])}
+      </td>
+    </tr>
+  ) : null;
+
+const RSReloadSchema = ({ readOnlyMode, remoteSchemaName, ...props }) =>
+  !readOnlyMode && remoteSchemaName ? (
+    <div className={`${styles.commonBtn} ${styles.detailsRefreshButton}`}>
+      <span>
+        <ReloadRemoteSchema {...props} remoteSchemaName={remoteSchemaName} />
+      </span>
+      <span>
+        <ToolTip
+          icon="fa-question-circle"
```

Comment:
We had a hardcoded icon in `Tooltip` for consistency. Can you revert this change in `Tooltip`?